WORKSPACE

```diff
@@ -60,14 +60,15 @@ load("//third_party/cargo_raze:repos.bzl", "raze_repos")
 raze_repos()
 load("//third_party/cargo_raze:deps.bzl", "raze_deps")
 raze_deps()
-# The raze instructions would have us call `cargo_raze_transitive_deps`, but that
-# wants to re-instantiate rules_rust and mess up our rust configuration.
-# Instead, we perform the single other action that transitive_deps would perform:
-# load and instantiate `rules_foreign_cc_dependencies`.
-#load("@cargo_raze//:transitive_deps.bzl", "cargo_raze_transitive_deps")
-#cargo_raze_transitive_deps()
-load("@rules_foreign_cc//foreign_cc:repositories.bzl", "rules_foreign_cc_dependencies")
-rules_foreign_cc_dependencies()
+# The raze instructions would have us call `cargo_raze_transitive_deps`, but
+# that wants to re-instantiate rules_rust and mess up our rust configuration.
+# The single other action that transitive_deps would perform is to load and
+# instantiate `rules_foreign_cc_dependencies`, but this has already been done
+# above, so we can do nothing here.
```

Comment:
This feels like it should be split into a separate pr... but now I see this is a draft

Reply:
Yep, this is a "dependent PR" — this first 3 commits will disappear when we merge #15853.

Reply:
Parent is merged. Ready for review!